javaext/com.microsoft.java.lombok/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,12 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Lombok extension
+Bundle-SymbolicName: com.microsoft.java.lombok;singleton:=true
+Bundle-Version: 0.1.0
+Bundle-Activator: com.microsoft.java.lombok.LombokPlugin
+Bundle-RequiredExecutionEnvironment: JavaSE-11
```

Comment:
do we need to update it to 17?

Reply:
yes, since it's a new extension, it's ok to update to 17 directly.